<!DOCTYPE html>
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8"/>
<meta content="SAS 9.4" name="generator"/>
<title>SAS Output</title>
<style>
/*<![CDATA[*/
.body.c > table, .body.c > pre, .body.c div > table,
.body.c div > pre, .body.c > table, .body.c > pre,
.body.j > table, .body.j > pre, .body.j div > table,
.body.j div > pre, .body.j > table, .body.j > pre,
.body.c p.note, .body.c p.warning, .body.c p.error, .body.c p.fatal,
.body.j p.note, .body.j p.warning, .body.j p.error, .body.j p.fatal,
.body.c > table.layoutcontainer, .body.j > table.layoutcontainer { margin-left: auto; margin-right: auto }
.layoutregion.l table, .layoutregion.l pre, .layoutregion.l p.note,
.layoutregion.l p.warning, .layoutregion.l p.error, .layoutregion.l p.fatal { margin-left: 0 }
.layoutregion.c table, .layoutregion.c pre, .layoutregion.c p.note,
.layoutregion.c p.warning, .layoutregion.c p.error, .layoutregion.c p.fatal { margin-left: auto; margin-right: auto }
.layoutregion.r table, .layoutregion.r pre, .layoutregion.r p.note,
.layoutregion.r table, .layoutregion.r pre, .layoutregion.r p.note,
.layoutregion.r p.warning, .layoutregion.r p.error, .layoutregion.r p.fatal { margin-right: 0 }
article, aside, details, figcaption, figure, footer, header, hgroup, nav, section { display: block }
html{ font-size: 100% }
.body { margin: 1em; font-size: 13px; line-height: 1.231 }
sup { position: relative; vertical-align: baseline; bottom: 0.25em; font-size: 0.8em }
sub { position: relative; vertical-align: baseline; top: 0.25em; font-size: 0.8em }
ul, ol { margin: 1em 0; padding: 0 0 0 40px }
dd { margin: 0 0 0 40px }
nav ul, nav ol { list-style: none; list-style-image: none; margin: 0; padding: 0 }
img { border: 0; vertical-align: middle }
svg:not(:root) { overflow: hidden }
figure { margin: 0 }
table { border-collapse: collapse; border-spacing: 0 }
.layoutcontainer { border-collapse: separate; border-spacing: 0 }
p { margin-top: 0; text-align: left }
h1.heading1 { text-align: left }
h2.heading2 { text-align: left }
h3.heading3 { text-align: left }
h4.heading4 { text-align: left }
h5.heading5 { text-align: left }
h6.heading6 { text-align: left }
span { text-align: left }
table { margin-bottom: 1em }
td, th { text-align: left; padding: 3px 6px; vertical-align: top }
td[class$="fixed"], th[class$="fixed"] { white-space: pre }
section, article { padding-top: 1px; padding-bottom: 8px }
hr.pagebreak { height: 0px; border: 0; border-bottom: 1px solid #c0c0c0; margin: 1em 0 }
.stacked-value { text-align: left; display: block }
.stacked-cell > .stacked-value, td.data > td.data, th.data > td.data, th.data > th.data, td.data > th.data, th.header > th.header { border: 0 }
.stacked-cell > div.data { border-width: 0 }
.systitleandfootercontainer { white-space: nowrap; margin-bottom: 1em }
.systitleandfootercontainer > p { margin: 0 }
.systitleandfootercontainer > p > span { display: inline-block; width: 100%; white-space: normal }
.batch { display: table }
.toc { display: none }
.proc_note_group, .proc_title_group { margin-bottom: 1em }
p.proctitle { margin: 0 }
p.note, p.warning, p.error, p.fatal { display: table }
.notebanner, .warnbanner, .errorbanner, .fatalbanner,
.notecontent, .warncontent, .errorcontent, .fatalcontent { display: table-cell; padding: 0.5em }
.notebanner, .warnbanner, .errorbanner, .fatalbanner { padding-right: 0 }
.body > div > ol li { text-align: left }
.beforecaption > h4 { margin-top: 0; margin-bottom: 0 }
.c { text-align: center }
.r { text-align: right }
.l { text-align: left }
.j { text-align: justify }
.d { text-align: right }
.b { vertical-align: bottom }
.m { vertical-align: middle }
.t { vertical-align: top }
.accessiblecaption {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.aftercaption {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.batch > col, .batch > colgroup > col, .batch > colgroup, .batch > tr, .batch > * > tr, .batch > thead, .batch > tbody, .batch > tfoot { border: none; }
.batch {
    background-color: #ffffff;
    border-style: solid;
    border-collapse: separate;
    border-spacing: 1px;
    color: #003399;
    font-family: 'SAS Monospace', 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    padding: 7px;
    }
.beforecaption {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.body {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    margin-left: 8px;
    margin-right: 8px;
}
.bodydate {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    text-align: right;
    vertical-align: top;
    width: 100%;
}
.bycontentfolder {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: none;
    margin-left: 6pt;
}
.byline {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.bylinecontainer > col, .bylinecontainer > colgroup > col, .bylinecontainer > colgroup, .bylinecontainer > tr, .bylinecontainer > * > tr, .bylinecontainer > thead, .bylinecontainer > tbody, .bylinecontainer > tfoot { border: none; }
.bylinecontainer {
    background-color: #ffffff;
    border: none;
    border-spacing: 1px;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.caption {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.cell, .container {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.contentfolder, .contentitem {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: none;
    margin-left: 6pt;
}
.contentproclabel, .contentprocname {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: decimal;
}
.contents {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: decimal;
    margin-left: 8px;
    margin-right: 8px;
}
.contentsdate {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.contenttitle {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.continued {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
    width: 100%;
}
.data {
    background-color: #ffffff;
    color: #000000;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.dataemphasis {
    background-color: #ffffff;
    color: #000000;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.dataemphasisfixed {
    background-color: #ffffff;
    color: #000000;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.dataempty {
    background-color: #ffffff;
    color: #000000;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.datafixed {
    background-color: #ffffff;
    color: #000000;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.datastrong {
    background-color: #ffffff;
    color: #000000;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.datastrongfixed {
    background-color: #ffffff;
    color: #000000;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.date {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.document, .errorbanner, .errorcontent {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.errorcontentfixed {
    background-color: #ffffff;
    color: #003399;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.extendedpage {
    background-color: #ffffff;
    border-style: solid;
    border-width: 1pt;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
    text-align: center;
}
.fatalbanner, .fatalcontent {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.fatalcontentfixed {
    background-color: #ffffff;
    color: #003399;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.folderaction {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.footer {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.footeremphasis {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.footeremphasisfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.footerempty {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.footerfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.footerstrong {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.footerstrongfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.frame {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.graph > col, .graph > colgroup > col, .graph > colgroup, .graph > tr, .graph > * > tr, .graph > thead, .graph > tbody, .graph > tfoot { border: none; }
.graph {
    background-color: #ffffff;
    border-style: solid;
    border-collapse: separate;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    }
.header {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.headeremphasis {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.headeremphasisfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.headerempty {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.headerfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.headersandfooters, .headerstrong {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.headerstrongfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.index, .indexaction {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.indexitem {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: none;
    margin-left: 6pt;
}
.indexprocname {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: decimal;
}
.indextitle, .linecontent, .list, .list10, .list2, .list3, .list4, .list5, .list6, .list7, .list8, .list9, .listitem, .listitem10, .listitem2, .listitem3, .listitem4, .listitem5, .listitem6, .listitem7, .listitem8, .listitem9, .note, .notebanner, .notecontent {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.notecontentfixed {
    background-color: #ffffff;
    color: #003399;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.output > col, .output > colgroup > col, .output > colgroup, .output > tr, .output > * > tr, .output > thead, .output > tbody, .output > tfoot { border: none; }
.output {
    background-color: #cccccc;
    border-style: solid;
    border-collapse: separate;
    border-spacing: 1px;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    }
.pageno {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
    text-align: right;
    vertical-align: top;
}
.pages {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: decimal;
    margin-left: 8px;
    margin-right: 8px;
}
.pagesdate {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.pagesitem {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: none;
    margin-left: 6pt;
}
.pagesproclabel, .pagesprocname {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    list-style-type: decimal;
}
.pagestitle, .paragraph {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.parskip {
    background-color: #ffffff;
    border-spacing: 0;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.prepage, .proctitle {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.proctitlefixed {
    background-color: #ffffff;
    color: #003399;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowfooter {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowfooteremphasis {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.rowfooteremphasisfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.rowfooterempty {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowfooterfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.rowfooterstrong {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowfooterstrongfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowheader {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowheaderemphasis {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.rowheaderemphasisfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: italic;
    font-weight: normal;
}
.rowheaderempty {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowheaderfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.rowheaderstrong {
    background-color: #6495ed;
    color: #ffffff;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.rowheaderstrongfixed {
    background-color: #6495ed;
    color: #ffffff;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: bold;
}
.systemfooter, .systemfooter10, .systemfooter2, .systemfooter3, .systemfooter4, .systemfooter5, .systemfooter6, .systemfooter7, .systemfooter8, .systemfooter9 {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: italic;
    font-weight: bold;
}
.systemtitle, .systemtitle10, .systemtitle2, .systemtitle3, .systemtitle4, .systemtitle5, .systemtitle6, .systemtitle7, .systemtitle8, .systemtitle9 {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: medium;
    font-style: normal;
    font-weight: bold;
}
.systitleandfootercontainer > col, .systitleandfootercontainer > colgroup > col, .systitleandfootercontainer > colgroup, .systitleandfootercontainer > tr, .systitleandfootercontainer > * > tr, .systitleandfootercontainer > thead, .systitleandfootercontainer > tbody, .systitleandfootercontainer > tfoot { border: none; }
.systitleandfootercontainer {
    background-color: #ffffff;
    border: none;
    border-spacing: 1px;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.table > col, .table > colgroup > col, .table > colgroup, .table > tr, .table > * > tr, .table > thead, .table > tbody, .table > tfoot { border: none; }
.table {
    background-color: #cccccc;
    border-style: solid;
    border-collapse: separate;
    border-spacing: 1px;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    }
.titleandnotecontainer > col, .titleandnotecontainer > colgroup > col, .titleandnotecontainer > colgroup, .titleandnotecontainer > tr, .titleandnotecontainer > * > tr, .titleandnotecontainer > thead, .titleandnotecontainer > tbody, .titleandnotecontainer > tfoot { border: none; }
.titleandnotecontainer {
    background-color: #ffffff;
    border: none;
    border-spacing: 1px;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
    width: 100%;
}
.titlesandfooters, .usertext, .warnbanner, .warncontent {
    background-color: #ffffff;
    color: #003399;
    font-family: Arial, Helvetica, sans-serif;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
.warncontentfixed {
    background-color: #ffffff;
    color: #003399;
    font-family: 'Courier New', Courier, monospace;
    font-size: x-small;
    font-style: normal;
    font-weight: normal;
}
/*]]>*/
</style>
</head>
<body class="c body">
<div style="padding-bottom: 8px; padding-top: 1px">
<div id="IDX" class="systitleandfootercontainer" style="border-spacing: 1px">
<p><span class="c systemtitle">GWAS - (genome wide association studies) data set 3</span> </p>
</div>
<div style="padding-bottom: 8px; padding-top: 1px">
<table class="table" style="border-spacing: 1px" aria-label="Data Set REF.GWAS_SORTED_3">
<caption aria-label="Data Set REF.GWAS_SORTED_3"></caption>
<colgroup><col/><col/><col/><col/></colgroup>
<thead>
<tr>
<th class="r header" scope="col">sub_ref_no</th>
<th class="header" scope="col">Strengths</th>
<th class="header" scope="col">Weaknesses</th>
<th class="header" scope="col">Future_Research</th>
</tr>
</thead>
<tbody>
<tr>
<td class="r data">1</td>
<td class="data"> &#160;</td>
<td class="data"> &#160;</td>
<td class="data">However, the interrelation between HDL lipidome, proteome and particle functionality is a missing part of the puzzle, which remains as yet unsolved.  XXXXXXXXXXXXXXXXXXXXXXXX</td>
</tr>
<tr>
<td class="r data">2</td>
<td class="data"> &#160;</td>
<td class="data"> &#160;</td>
<td class="data">One necessary future development is post-selection inference (31, 32) in the high-dimensional multivariable MR framework. MR-BMA does not provide unbiased causal effect estimates. Re-fitting an unbiased multivariable MR model after risk factor selection would ignore the uncertainty of the selection and consequently not provide valid inferences. XXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
</tr>
<tr>
<td class="r data">3</td>
<td class="data"> &#160;</td>
<td class="data">We are aware of certain limitations to our study. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX First, due to differences in imputation reference panels between the studies included in the GWAS for glycine, sample sizes differed between the tested variants. Therefore certain loci, in particular those only covered in UK10K and rare variants, which have been shown to influence amino acid levels (13), may not have reached significance. XXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Secondly, some of the studies upon which our analyses were based were enriched for healthy participants, e.g., UK Biobank (55) and INTERVAL. It has recently been suggested that selection bias could theoretically lead to a false positive genetic association between an exposure and an outcome if both the exposure and the outcome influence the likelihood of an individual participating in the study (56). As individuals are not aware of their glycine levels, any such selection would have to be indirect through other factors that affect glycine and bias participation in each of the independent studies included in the same direction. We therefore consider the extent to which collider bias could be influencing the MR results due to a healthy participant effect to be small. XXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Finally, despite undertaking sensitivity analyses, we cannot exclude the possibility that our findings may be driven by vertical pleiotropic effects of the genetic instruments. XXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX As the metabolome is comprised of thousands of metabolites connected through numerous reactions, some of which may be unknown, the distinction between horizontal and vertical pleiotropy is difficult to make in the context of metabolomics. XX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX The 2 SNP score, comprised of loci in genes of major enzymes of glycine catabolism, is the most specific score to the glycine pathway but still shows modest associations with some metabolites of which we cannot be sure that they are metabolically linked to glycine, e.g., the unknown metabolite X-16570, and with glycine-conjugated metabolites, which may have effects independent of glycine on cardiometabolic disease risk.  XXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Therefore, we cannot fully exclude that glycine-independent mechanisms may have biased the genetically predicted association of glycine with disease risk. Moreover, as glycine is a metabolite on the intersection of many metabolic pathways, the genetic association of glycine with cardio-metabolic diseases may represent the causal effect of a metabolite to which glycine is metabolically close linked (e.g., tetrahydrofolate or serine).XXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
<td class="data"> &#160;</td>
</tr>
<tr>
<td class="r data">4</td>
<td class="data"> &#160;</td>
<td class="data">However =&gt; one HRC limitation is the absence of indels: these make up 4% of total variants in the phase 3 1000G reference panel (18)XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX The present study was limited to individuals of European ancestry: integration of these data with large-scale GWAS data from other major ancestral groups (as is being pursued by the DIAMANTE consortium) is expected to provide an additional boost to locus discovery, and to support further increases in causal variant resolution, most obviously at loci where extensive LD within European subjects limits the power of fine-mapping XXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
<td class="data">Both sites are present in the Genome Aggregation Database (28) and met quality control criteria in our data (average imputation quality &gt;0.7; association signal visible in multiple contributing studies), but their precise contribution to T2D-risk requires further in vitro and in vivo validation (THE CONTRIBUTION OF LOWER-FREQUENCY VARIANTS). XXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Altered glycation has the potential to impact on multiple processes, and global perturbation of ST6GAL1 has broad effects including, in St6gal1 knockout mice =&gt; increased body weight and visceral fat accumulation (41).</td>
</tr>
<tr>
<td class="r data">5</td>
<td class="data"> &#160;</td>
<td class="data">A general challenge for the study of genetic determinants of the human metabolome is how to integrate these large data sets and to synthesize, interpret, and display the generated wealth of information. XXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Current genetic data sets based on standard genome-wide arrays (Whole-genome genotyping microarrays have been used to successfully identify regions of the human genome that contribute to disease susceptibility and phenotypes) followed by state-of-the-art genotype imputation contain approximately 10 million common and well-imputed variants for individuals of European ancestry. XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Current non-targeted MS-based metabolomics platforms such as the services offered by Metabolon (Durham, North Carolina) return quantitative information for more than 1,500 metabolites in biological fluids. Hence, the study of the genetic basis of the individual metabolites and their pair-wise ratios (Many comparative functional genomic studies rely on pairwise comparisons, where traits of each gene are compared with traits of other genes across species. This leads to many more comparisons than unique observations, making each comparison dependent on others) requires the conduct of 1,500 and (1, 124, 250) GWAS (A genome-wide association study (abbreviated GWAS) is a research approach used to identify genomic variants that are statistically associated with a risk for a disease or a particular trait), respectively, in one biofluid alone. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Various methods such as OmicABLE and MatrixEQTL have been developed to accelerate the computation of linear models in a GWAS setting. (48, 49) Because the number of genome-wide significant results from well-powered mGWAS typically is high, the processing, aggregation, and annotation of results for further interpretation requires automated procedures. In recent mGWAS by Shin et al (8) and Raffler et al, (27) gene assignment within a locus and determination of the overlap of associated single-nucleotide polymorphisms (SNPs) with previous mGWAS and GWAS with clinical end points, was facilitated through automated combination of complementary sources of evidence. XXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  For this task, Raffler et al (27) made use of the SNiPA resource, (50) which provides SNP-centric information integrated from various gene- and SNP-based annotation data sets such as EnsEMBL, (51) GTEx, (52) GWAS Catalog, (53) and others together with pre-calculated linkage disequilibrium (Linkage Disequilibrium (LD) is a term coined to describe the non-random association between alleles of genetic loci that lie in proximity to each other.) data for all super-populations (superpopulation (plural superpopulations) The larger population from which a subpopulation is drawn) from the 1000 Genomes Project. (54) Because SNiPA allows for easy access to 500,000 metabolite quantitative trait loci (QTLs), 15,000 protein QTLs, 20 million expression QTLs, 400,000 regulatory clusters, 600,000 microRNA target sites, 1 million gene-associated enhancers and promoters, and 330,000 disease and phenotype associations, we also used the resource for extracting information for Table 2, Table 3 in this review. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Another challenge for the interpretation of findings from mGWAS is that a large part of the metabolome is still uncharted territory, with new metabolites discovered on a weekly basis. This is illustrated by the services offered by Metabolon: while Suhre et al7 reported on 276 blood metabolites in 2011, a recent study by Long et al9 using an update of the platform already reported on 486 identified metabolites. In addition, the studied metabolites included 158 unnamed molecules that can be recognized and quantified (named X-*) but whose identity has yet to be confirmed using isotope-labeled standards. These unknown metabolites can represent hitherto-unknown intermediates or products of human metabolism, or can be of xenobiotic origin, including medications, toxins, or products from microbiota or food. Despite major advances in metabolite identification over the past years through the development of chemoinformatic tools that systematically use and integrate spectral properties of unknown metabolites, annotation of as yet unidentified signals in non-targeted metabolomics data sets remains a huge challenge in the field. (55) As shown for both NMR- and MS-based non-targeted metabolomics, the functions of genes that are associated with unnamed compounds can provide complementary biological insights that can aid in metabolite identification. (26, 56, 57) XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX By combining bioinformatics and chemoinformatic approaches (eg, using knowledge about the implicated geneâ€™s function together with molecule-specific spectral characteristics such as mass and retention time), (58) it may be possible to identify novel and previously unsuspected molecules of importance in the etiology and/or progression of kidney diseases. Published mGWAS used data from a variety of platforms. Although many GIMs could be replicated across platforms, other GIMs were uniquely identified on particular platforms. Even for the same metabolite, differences in metabolite quantification as in targeted versus non-targeted methods can result in a subset of associations that are unique to each method. As an example, Yet et al (59) investigated how a targeted (Biocrates (Innsbruck, Austria) panel) and an non-targeted (Metabolon) MS-based method for metabolite quantification compared in mGWAS performed in the same study population. XXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Genetic associations for 43 known metabolites present on both platforms generally correlated well. There were 18 loci unique to the Metabolon platform and 1 locus unique to the Biocrates platform, which underscores the complementary nature of different platforms and the potential of non-targeted metabolomics. The latter also was shown in the urine mGWAS by Raffler et al,27 who described genetic loci uniquely identified using a non-targeted approach or a targeted approach based on the same measured spectra. Even if the identity of a metabolite is known and it is quantified from the same biofluid and was measured on the same metabolomics platform, the lack of standardization across published mGWAS studies complicates a comparison of reported effect sizes. Differences result from the methods used for data cleaning, imputation, trait transformations, and data analysis. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Studies of patients with CKD raise additional questions: in addition to low eGFR, these patients also often manifest considerable proteinuria. Depending on the metabolomics approach used, this may influence results from mGWAS studies in urine. Although NMR-based methods quantify the amount of free metabolite concentrations, the quantified fraction of total metabolite concentration may vary depending on the specific procedure of sample preparation (eg, solvent used, and so forth) that is used in an MS-based method. This may affect the ability to identify genetic associations with metabolites that display a high degree of protein binding such as tryptophan. It is conceivable that free metabolite concentrations are more relevant for the identification of metabolite transporters along the nephron, although a direct comparison of genetic studies of metabolites quantified via both NMR and MS in the same samples is lacking.  XXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX In most mGWAS, interrelation between metabolites within pathways are ignored; only a few studies have used the correlation between metabolites within pathways for their analysis. (60, 61) In some studies, the investigators inferred metabolic networks based on partial correlations of metabolite levels, which have been shown to reconstruct true biochemical relationships, to embed the identified genetic association into their metabolic context for aiding biological interpretation. (8, 56, 62, 63)  XXXXXXXXXXXXXXXXXXXXXXXXX  XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
<td class="data">Extending this concept to infer higher-dimensional relationships between metabolites (in multiple fluids), proteins, and genes across omics layers is currently limited by the availability of such multi-omics data for the same samples, but might provide useful additional insights for kidney function in the future. (64, 65) XXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Future efforts in the mGWAS area should aim to improve comparability across studies through standardization of analytic approaches. In the absence of a unified technology that allows for a comprehensive and standardized quantification of all metabolites in a sample, imputation of missing metabolites may be a fruitful area of future research. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Our understanding of the influence of disease states and environmental influences, that is, the dynamic aspect of the metabolome, also should be a major focus of future studies. XXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX In summary, the integration of genomic and metabolomic data is an exciting research area for the field of nephrology, which we expect to grow significantly in the coming years. XXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
</tr>
<tr>
<td class="r data">6</td>
<td class="data"> &#160;</td>
<td class="data"> &#160;</td>
<td class="data">The current results emphasize that progressively larger sample sizes, coupled with higher density sequence-based imputation (13), will continue to represent a powerful strategy for genetic discovery in T2D and in complex diseases and traits more generally. At known T2D-associated loci, identification of the most plausible T2D causal variants will likely require large-scale multiethnic analyses, where more diverse haplotypes, reflecting different patterns of LD, in combination with functional (31,50,51) data allow refinement of association signals to smaller numbers of variants (2) XXXXXXXXXXXXXXXXXXXXXXXX</td>
</tr>
</tbody>
</table>
</div>
</div>
</body>
</html>
